xcdat/dataset.py

```diff
@@ -308,16 +313,24 @@ def decode_non_cf_time(dataset: xr.Dataset) -> xr.Dataset:
     time_bounds = ds.get(time.attrs.get("bounds"), None)
     units_attr = time.attrs.get("units")
 
-    # If the time units cannot be split into a unit and reference date, it
-    # cannot be decoded so the original dateset is returned.
     try:
         units, ref_date = _split_time_units_attr(units_attr)
     except ValueError:
+        logger.warning(
+            "Attempted to decode non-CF compliant time coordinates, but the units "
+            f"({units}) is not in a supported format ('months since...' or "
+            "'months since...'). Returning existing time coordinates."
+        )
         return ds
```

Comment:
```suggestion
    try:
        units, ref_date = _split_time_units_attr(units_attr)
    except ValueError:
        logger.warning(
            "Attempted to decode non-CF compliant time coordinates, but the units "
            f"({units}) is not in a supported format ('months since...' or "
            "'months since...'). Returning dataset with existing time coordinates."
        )
        return ds
```